### PR TITLE
feat(webhooks): allow proxy settings for webhook notifications

### PR DESCRIPTION
**Pull request by RomanJRW**
relates to OPT-6083

### CODE REVIEW

**Comment by RomanJRW** [comment]
@akeller - done 👍 
Let's hold off on merging before https://github.com/camunda/camunda-optimize/pull/4903 gets approved too if that's ok?

File: versioned_docs/version-1.3/self-managed/optimize-deployment/setup/configuration.md [hunk 1]
```diff
@@ -305,7 +305,10 @@ Settings for webhooks which can receive custom alert notifications. You can conf
 |webhookAlerting.webhooks.${webhookName}.headers||A map of the headers of the request to be sent to the webhook.|
 |webhookAlerting.webhooks.${webhookName}.httpMethod||The HTTP Method of the request to be sent to the webhook.|
 |webhookAlerting.webhooks.${webhookName}.defaultPayload||The payload of the request to be sent to the webhook. This should include placeholder keys that allow you to define dynamic content. See [Alert Webhook Payload Placeholders](../webhooks#alert-webhook-payload-placeholders) for available values.|
-
+|webhookAlerting.webhooks.${webhookName}.proxy.enabled|false|Whether an HTTP proxy should be used for requests to the webhook URL.|
```
Comment (grlimacan) [inline versioned_docs/version-1.3/self-managed/optimize-deployment/setup/configuration.md:308]:
```suggestion
|webhookAlerting.webhooks.${webhookName}.proxy.enabled|false|Whether a HTTP proxy should be used for requests to the webhook URL.|
```

**Review by grlimacan — DISMISSED** [review]
Generally ok, but some small things need to be corrected, pls take a look at the comments

File: versioned_docs/version-1.3/self-managed/optimize-deployment/setup/configuration.md [hunk 2]
```diff
@@ -305,7 +305,10 @@ Settings for webhooks which can receive custom alert notifications. You can conf
 |webhookAlerting.webhooks.${webhookName}.headers||A map of the headers of the request to be sent to the webhook.|
```
[the remark this answers is not in the document]
Comment (RomanJRW) [reply versioned_docs/version-1.3/self-managed/optimize-deployment/setup/configuration.md:305]: can you double check this? My IntelliJ renderer also looks broken like you describe, but in any other markdown renderer I think it's fine. Did you see it broken on the actual docs page?

**Comment by RomanJRW** [comment]
@akeller - is this good to merge now?

File: versioned_docs/version-1.3/self-managed/optimize-deployment/setup/configuration.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (akeller) [reply versioned_docs/version-1.3/self-managed/optimize-deployment/setup/configuration.md:305]: I'll check this before merging.

[the remark this answers is not in the document]
Comment (akeller) [reply versioned_docs/version-1.3/self-managed/optimize-deployment/setup/configuration.md:305]: This looks good to me after doing an `npm run start` locally. This table appears to have no default values, which is fine.

**Pull request merged**